app/src/main/res/values/strings.xml

```diff
@@ -53,6 +53,7 @@
   <string name="featured_questions">Featured Questions</string>
   <string name="frequently_asked_questions">Frequently Asked Questions</string>
   <string name="faq_activity_title">FAQs (Frequently Asked Questions)</string>
+  <string name="pin_password_title">PIN verification</string>
```

Comment:
Rename this to `pin_password_activity_title`

Reply:
Done!!